roles/st2chatops/vars/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+supported_hubot_adapter:
+ - slack
+ - hipchat
+ - xmpp
+ - flowdock
+ - yammer
+ - spark
+ - irc
```

Comment:
Please also include `shell` adapter: https://github.com/github/hubot/blob/master/docs/adapters/shell.md

Do you think it will be better if we use `shell` adapter instead of Slack for testing/smoketests purposes in TravisCI?